packages/disputer/index.js

```diff
@@ -89,7 +88,6 @@ async function run({
       );
 
     // Setup contract instances.
-    const voting = new web3.eth.Contract(getAbi("Voting", "1.2.2"), getAddress("Voting", networkId));
```

Comment:
No more need to pass in `voting` to Disputer, which was previously used to make a `getPrice` call to figure out the settlement price. `settlementPrice` is part of event params!

Reply:
Might be a dumb question, but won't we still need it if we're interacting with an older EMP?

Reply:
Ah, saw your comment below about changing this a while back.